package.json

```diff
@@ -3,16 +3,31 @@
   "version": "1.0.0",
   "description": "Basic Stream Deck hardware emulator",
   "main": "index.js",
+  "bin": {
+    "sde": "./bin/sde"
+  },
   "scripts": {
     "start": "node index.js",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "",
   "license": "MIT",
+  "homepage": "https://github.com/FritzAndFriends/StreamDeckEmulator",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com/FritzAndFriends/StreamDeckEmulator.git"
```

Comment:
Just noticed this trying to see how I might need to merge changes I'm trying, and for the repo URL, I wonder if we might need the **https** link, rather than SSH, from what I see on the [npmjs.org](https://www.npmjs.org) site. For example, on the [.env](https://www.npmjs.org/package/dotenv) package. Unless the site knows how to translate to/from the different formats based on needs?

Reply:
Here are the [relevant sections](https://github.com/motdotla/dotenv/blob/master/package.json#L13-L15) of the dotenv package and the npm [spec for package.json](https://docs.npmjs.com/files/package.json#repository).

It looks like maybe the best option is:

```
"repository": "github:user/repo"
```

I'll update.

Reply:
Wow, they've certainly got a breadth of conventions which can be followed!